operator/builtin/transformer/move/testdata/MoveBodyToAttribute.yaml

```diff
@@ -0,0 +1,3 @@
+type: move
+from: $body.key
```

Comment:
Can you add a couple tests where `$body` is omitted from the value? Since `$body` is the implied prefex, we should test that is works without it. It would be good to have at least one test for each of `from` and `to`.